src/condor_utils/file_transfer.cpp

```diff
@@ -4528,6 +4535,7 @@ int FileTransfer::InvokeMultipleFileTransferPlugin( CondorError &e,
 	// Determine if we want to run the plugin with root priv (if available).
 	// If so, drop_privs should be false.  the default is to drop privs.
 	bool drop_privs = !param_boolean( "RUN_FILETRANSFER_PLUGINS_WITH_ROOT", false );
+	if (plugins_from_job.find(plugin_path) != plugins_from_job.end()) { drop_privs = true; }
```

Comment:
Below you use the `count()`; I actually like that form better:
```suggestion
	if (plugins_from_job.count(plugin_path)) { drop_privs = true; }
```

Sigh.  The `contains()` method doesn't make it until C++20 ... I assume we'll be able to use it in a decade or so!